mantidimaging/gui/widgets/spectrum_widgets/fitting_display_widget.py

```diff
@@ -67,11 +67,10 @@ def set_default_region(self, x_data: np.ndarray, y_data: np.ndarray) -> None:
         """Position the ROI centrally over the plotted data."""
         x_min, x_max = float(np.nanmin(x_data)), float(np.nanmax(x_data))
         y_min, y_max = float(np.nanmin(y_data)), float(np.nanmax(y_data))
-        x_span = max((x_max - x_min) * 0.25, 20.0)
-        y_span = max((y_max - y_min) * 0.5, 10.0)
+        x_span = max((x_max - x_min) * 0.25, (x_max - x_min) * 0.05)
+        y_span = max((y_max - y_min) * 0.5, (y_max - y_min) * 0.05)
```

Comment:
I think this could just be
```python
x_span = (x_max - x_min) * 0.25
y_span = (y_max - y_min) * 0.5
```
Maybe make them both 0.25 for consistency